theano/tensor/tests/test_extra_ops.py

```diff
@@ -705,7 +705,7 @@ def test_basic_matrix(self):
         # Done by using the op and checking that it returns the right answer.
 
         x = theano.tensor.matrix()
-        inp = np.asarray([[2, 1], [3, 2], [2, 3]], dtype=config.floatX)
+        inp = np.asarray([[2, 1], [3, 2], [2, 1]], dtype=config.floatX)
```

Comment:
Why was this required?

Reply:
Just to have a duplicate row. Numpy returns a 2x2 array when the `axis=0` parameter value is supplied, and a 3x2 array when `axis=-1`. I wanted to test that the `Op` did this too.